submissions/iakow/html-css-popup/style.css

```diff
@@ -0,0 +1,254 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+body {
+  height: 100vh;
+  background-image: url(./img/boys2.png);
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+nav {
+  padding: 5px 0;
+  background-color: rgb(51, 51, 51);
+}
+
+nav>ul {
+  display: flex;
+  justify-content: center;
+}
+
+.relative-box {
+  position: relative;
+}
+
+.nav-item {
+  display: block;
+  height: 40px;
+  width: 40px;
+  margin: 0 5px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  cursor: pointer;
+}
+
+.nav-item:hover {
+  background-color: rgb(98 98 98);
+}
+
+.popup-ctrl, .popup-expansion {
```

Comment:
It's better to use these styles for hiding `input`:
``` CSS
.visually-hidden {
	position: absolute !important;
	clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
	clip: rect(1px, 1px, 1px, 1px);
	padding:0 !important;
	border:0 !important;
	height: 1px !important; 
	width: 1px !important; 
	overflow: hidden;
}
```
More info [here](https://htmlacademy.ru/blog/boost/frontend/short-12)

Reply:
Thanks! Done.